x509-store/Data/X509/CertificateStore.hs

```diff
@@ -26,12 +31,22 @@ import qualified Data.ByteString as B
 data CertificateStore = CertificateStore (M.Map DistinguishedName SignedCertificate)
                       | CertificateStores [CertificateStore]
 
+#if MIN_VERSION_base(4,9,0)
+instance Semigroup CertificateStore where
+    (<>) = append
```

Comment:
why not
``` haskell
    (<>) = mappend
```
Then you won't need the outlining if `append`...

Reply:
as I understand mappend is disappearing from Monoid in the future, and being replaced by a simple `mappend = <>`

Reply:
I see. Thanks for the info!